### PR TITLE
tests: Address comments of #96

### PR DESCRIPTION
**Pull request by jcanocan**
**What this PR does / why we need it**:
It fixes concerns posted on https://github.com/kubevirt/kubevirt.core/pull/96

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes # [CNV-41852](https://issues.redhat.com/browse/CNV-41852)

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: tests/unit/plugins/inventory/test_kubevirt.py [hunk 1]
```diff
@@ -155,7 +155,7 @@
     }
 )
 LOADBALANCER_SERVICE_WITHOUT_SELECTOR_AND_SSH_PORT = merge_dicts(
```
Comment (0xFelix) [inline tests/unit/plugins/inventory/test_kubevirt.py:157]: Either use `LOADBALANCER` or `LB` everywhere?

File: tests/unit/plugins/inventory/test_kubevirt.py [hunk 2]
```diff
@@ -786,12 +775,12 @@ def inventory_groups_create_groups_option(inventory_groups):
 
 
 @pytest.fixture(scope="module")
-def base_vmi_host_vars():
+def empty_host_vars():
```
Comment (0xFelix) [inline tests/unit/plugins/inventory/test_kubevirt.py:778]: Pass something like `None` instead and check in the test that when `None` then it should be empty? Saves adding this fixture.

File: tests/unit/plugins/inventory/test_kubevirt.py (hunk 1 above)
Comment (jcanocan) [reply tests/unit/plugins/inventory/test_kubevirt.py:157]: Let's stick with the `LOADBALANCER` option.

File: tests/unit/plugins/inventory/test_kubevirt.py (hunk 2 above)
Comment (jcanocan) [reply tests/unit/plugins/inventory/test_kubevirt.py:778]: I would like to keep it as it is. While it is true that we are adding a "silly" fixture, IMHO the benefit is that it is really clear from the parametrize mark that the inventory should be empty without making the code harder to read (no if else statements required) the test code more.

File: tests/unit/plugins/inventory/test_kubevirt.py [hunk 3]
```diff
@@ -6,10 +6,11 @@
 
 __metaclass__ = type
 
+import copy
```
Comment (0xFelix) [inline tests/unit/plugins/inventory/test_kubevirt.py:9]: Add newline between? copy is stdlib, pytest is not :)

File: tests/unit/plugins/inventory/test_kubevirt.py (hunk 2 above)
Comment (0xFelix) [reply tests/unit/plugins/inventory/test_kubevirt.py:778]: It's okay in this case, but we really need to keep `test_kubevirt.py` maintainable.

File: tests/unit/plugins/inventory/test_kubevirt.py (hunk 3 above)
Comment (jcanocan) [reply tests/unit/plugins/inventory/test_kubevirt.py:9]: Done.

**Review by 0xFelix — APPROVED** [review]
Let's get this one merged and take the discussion of splitting up `test_kubevirt.py` to #97.

/lgtm
/approve

**Pull request merged**